### PR TITLE
test: init with jest

### PR DESCRIPTION
**Pull request by himself65**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,16 +30,19 @@
     "tslib": "^2.2.0"
   },
   "devDependencies": {
+    "@peculiar/webcrypto": "^1.1.7",
```
Comment (Jack-Works) [inline package.json:33]: Would you like to try this https://nodejs.org/api/webcrypto.html?

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 14.x
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14.x
```
Comment (Jack-Works) [inline .github/workflows/test.yml]:
```suggestion
          node-version: 16.x
```

File: api/util.ts [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import { decode } from 'cbor-redux'
+import type { AttestationObject } from '../types/interface'
+import { Buffer } from 'buffer'
+
+export function concatenate (...arrays: ArrayBuffer[]): ArrayBuffer {
```
Comment (Jack-Works) [inline api/util.ts]: Did you format with prettier? 🤔

File: .github/workflows/test.yml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 14.x
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16.x
+      - uses: actions/cache@v2
+        with:
+          path: ~/.pnpm-store
+          key: ${{ runner.os }}-node-${{ hashFiles('**/pnpm-lock.yaml') }}
```
Comment (Jack-Works) [inline .github/workflows/test.yml]:
```suggestion
          key: ${{ runner.os }}-node-${{ hashFiles('pnpm-lock.yaml') }}
```

File: .github/workflows/test.yml [hunk 5]
```diff
@@ -0,0 +1,18 @@
+name: Test
+on: [ push, pull_request ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 14.x
```
Comment (Jack-Works) [inline .github/workflows/test.yml]:
```suggestion
      - name: Use Node.js
```

File: package.json (hunk 1 above)
Comment (Jack-Works) [reply package.json:33]: Did you try it?

Comment (himself65) [reply package.json:33]: NO, the `SubtleCrypto` is added in NODEJS **15.0.0**

Comment (Jack-Works) [reply package.json:33]: So what's wrong? Can't you just use Node 15 or 16?

**Review by Jack-Works — APPROVED** [review]

**Pull request merged**